src/app/search/search.component.ts

```diff
@@ -410,6 +420,40 @@ export class SearchComponent implements OnInit, OnDestroy {
       .catch((error) => console.log(error));
   }
 
+  /**
+   * Updates the results table when there is no search term
+   * Queries for 100 tools and 100 workflows
+   *
+   * @param {string} value the elastic search query
+   * @memberof SearchComponent
+   */
+  updateResultsTableSeparately(toolsQuery: string, workflowsQuery: string) {
+    let toolHits;
+    let workflowHits;
+    // Combine array of tools and workflows to filter into results table
+    ELASTIC_SEARCH_CLIENT.search({
+      index: 'tools',
+      type: 'entry',
+      body: toolsQuery,
+    })
+      .then((tools) => {
+        toolHits = tools.hits.hits;
+        ELASTIC_SEARCH_CLIENT.search({
+          index: 'tools',
+          type: 'entry',
+          body: workflowsQuery,
+        })
+          .then((workflows) => {
+            workflowHits = workflows.hits.hits;
+            this.hits = toolHits.concat(workflowHits);
```

Comment:
The two calls are now sequentially ran which should work (but not ideal)

Reply:
I'd never paid attention to this client before -- is it returning a Promise? If so, I think you can create Observables from them, then use `forkJoin`.